mix.exs

```diff
@@ -62,9 +62,21 @@ defmodule Jellyfish.MixProject do
       {:protobuf, "~> 0.12.0"},
 
       # Membrane deps
-      {:membrane_rtc_engine, "~> 0.17.1"},
-      {:membrane_rtc_engine_webrtc, "~> 0.3.0"},
-      {:membrane_rtc_engine_hls, "~> 0.2.1"},
+      {:membrane_rtc_engine,
+       github: "jellyfish-dev/membrane_rtc_engine",
+       branch: "feature/cleanup",
+       sparse: "engine",
+       override: true},
+      {:membrane_rtc_engine_webrtc,
+       github: "jellyfish-dev/membrane_rtc_engine",
+       branch: "feature/cleanup",
+       sparse: "webrtc",
+       override: true},
+      {:membrane_rtc_engine_hls,
+       github: "jellyfish-dev/membrane_rtc_engine",
+       branch: "feature/cleanup",
+       sparse: "hls",
+       override: true},
```

Comment:
Merge https://github.com/jellyfish-dev/membrane_rtc_engine/pull/324 first and use master branch